build.sbt

```diff
@@ -255,6 +255,7 @@ lazy val scalahostSbt = project
     buildInfoSettings,
     sbt.ScriptedPlugin.scriptedSettings,
     sbtPlugin := true,
+    publishMavenStyle := false, // necessary for pre-releases to work with addSbtPlugin
```

Comment:
@xeno-by I couldn't find where publishTo is set to sonatype for stable releases, this field should only be false when publishing to Bintray.